src/quo2/components/wallet/wallet_activity/view.cljs

```diff
@@ -0,0 +1,169 @@
+   :buy     [i18n/label :t/buy]
+   :destroy [i18n/label :t/destroy]
+   :mint    [i18n/label :t/mint]})
+
+(def transaction-icon
+  {:receive :i/receive
+   :send    :i/send
+   :swap    :i/swap
+   :bridge  :i/bridge
+   :buy     :i/buy
+   :destroy :i/destroy
+   :mint    :i/mint})
+
+(def status-icon
+  {:pending   :i/pending-state
+   :confirmed :i/positive-state
+   :finalised :i/diamond-blue
+   :failed    :i/negative-state})
+
+(defn transaction-header
+  [{:keys [transaction
+           timestamp
+           counter
+           theme
+           blur?]
+    :or   {transaction :receive
+           counter     1}}]
+
+  [rn/view
+   {:style style/transaction-header-container}
+   [text/text
+    {:weight :semi-bold
+     :size   :paragraph-1
+     :style  (style/transaction-header theme)}
+    (transaction transaction-translation)]
+   (when (> counter 1)
+     [rn/view (style/transaction-counter-container theme blur?)
+      [text/text
+       {:weight :medium
+        :size   :label
+        :style  (style/transaction-counter theme)}
+       (str "x" counter)]])
+   [rn/view
+    [text/text
+     {:weight :regular
+      :size   :label
+      :style  (style/timestamp theme blur?)}
+     timestamp]]])
+
+(defn transaction-icon-view
+  [{:keys [theme blur? transaction status]
+    :or   {transaction :receive
+           status      :pending}}]
+  [rn/view {:style style/icon-container}
+   [hole-view/hole-view
+    {:style (style/icon-hole-view theme blur?)
+     :holes [{:x            20
+              :y            20
+              :right        0
+              :width        12
+              :height       12
+              :borderRadius 6}]}
+    [icon/icon (transaction-icon transaction)
+     {:color (style/icon-color theme)}]]
+   [rn/view {:style style/icon-status-container}
+    [icon/icon (status-icon status)
+     {:size     12
+      :no-color :true}]]])
+
+(defn prop-text
+  [label theme]
+  [text/text
+   {:weight :regular
+    :size   :paragraph-2
+    :style  (style/prop-text theme)}
+   [i18n/label label]])
+
+(defn prop-tag
+  [props blur?]
+  [rn/view {:style {:margin-right 4}}
+   [context-tag/view (merge props {:size 24 :blur? blur?})]])
+
+(defn- view-internal
+  [_]
+  (let [pressed? (reagent/atom false)]
+    (fn
+      [{:keys [state theme blur?
+               on-press
+               first second third fourth
+               second-prefix third-prefix fourth-prefix]
+        :as   props}]
+      [rn/pressable
+       {:style               (style/wallet-activity-container {:pressed? @pressed?
+                                                               :theme    theme
+                                                               :blur?    blur?})
+        :accessibility-label :wallet-activity
+        :disabled            (= state :disabled)
+        :on-press            on-press
+        :on-press-in         (fn [] (reset! pressed? true))
+        :on-press-out        (fn [] (reset! pressed? false))}
```

Comment:
You can use `#()` syntax here

Reply:
@briansztamfater, some devs prefer to avoid `#()` when there are no arguments being used. I don't always follow this idea, but I do tend to avoid `#()` in the cases where there's no function argument because this clearly communicates to the reader that they don't need to "search" for a `%` symbol somewhere. When I see `#()` I always search for `%` just to find out it's not there many times.

Some devs actually hate `#()` and others love it and request such a syntax in other Lisps. It's funny 🤷🏼